tests/e2e/specs/shopper/mini-cart.test.js

```diff
@@ -3,6 +3,8 @@
  */
 import { setDefaultOptions, getDefaultOptions } from 'expect-puppeteer';
 import { SHOP_CART_PAGE, SHOP_CHECKOUT_PAGE } from '@woocommerce/e2e-utils';
+const WooCommerceRestApi = require( '@woocommerce/woocommerce-rest-api' )
```

Comment:
Why not

```suggestion
import WooCommerceRestApi from '@woocommerce/woocommerce-rest-api';
```

Reply:
Fixed in [ff3ece3](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5979/commits/ff3ece3d3702e9e700cf5fcc75952d178ade3701)